demos/CMakeLists.txt

```diff
@@ -206,6 +206,7 @@ else()
 endif()
 
 find_package(ngraph REQUIRED)
+find_package(TBB REQUIRED tbb)
```

Comment:
@shubha-ramani cold you explain the purpose of that line?

We should be careful to use TBB from IE and not from Yocto. Can you confirm that TBB is taken from IE?

Reply:
@asomsiko, IE depends on TBB. User should pick right version of TBB by setting TBB_DIR env variable. If the user wants to cross compile samples then they should set TBB_DIR to point to Yocto TBB. In case of IA compilation TBB_DIR should be set to point to TBB version which comes with OpenVINO or just omitted.